SimPEG/EM/Static/SIP/ProblemSIP.py

```diff
@@ -1,59 +1,68 @@
-from SimPEG import Problem, Utils, Maps, Mesh
+import numpy as np
+
+from SimPEG import Utils
+from SimPEG import Props
+
 from SimPEG.EM.Base import BaseEMProblem
 from SimPEG.EM.Static.DC.FieldsDC import FieldsDC, Fields_CC, Fields_N
-from SimPEG.Utils import sdiag
-import numpy as np
-from SimPEG.Utils import Zero
 from SimPEG.EM.Static.DC import getxBCyBC_CC
 from .SurveySIP import Survey, Data
 
-class ColeColePropMap(Maps.PropMap):
-    """
-        Property Map for EM Problems. The electrical conductivity (\\(\\sigma\\)) is the default inversion property, and the default value of the magnetic permeability is that of free space (\\(\\mu = 4\\pi\\times 10^{-7} \\) H/m)
-    """
 
-    eta = Maps.Property("Electrical Conductivity", defaultInvProp=True)
-    tau = Maps.Property("Electrical Conductivity", defaultVal=0.1, propertyLink=('taui', Maps.ReciprocalMap))
-    taui   = Maps.Property("Electrical Conductivity", defaultVal=1., propertyLink=('tau', Maps.ReciprocalMap))
-    c   = Maps.Property("Electrical Conductivity", defaultVal=1.)
+class BaseSIPProblem(BaseEMProblem):
```

Comment:
do we want to reference ColeCole specifically in the problem name? This implementation looks specific to the ColeCole parametrization of conductivity